src/scancode/cli.py

```diff
@@ -1202,7 +1202,14 @@ def display_summary(codebase, scan_names, processes, verbose):
                                 '%(final_size_count)s' % locals())
 
     echo_stderr('Timings:')
+
+    timestamp = codebase.timings.get('timestamp',None)
+    timestamp = tstamp2time(timestamp)
+    echo_stderr('  timestamp: %(timestamp)s'% locals())
+    
     for name, value, in codebase.timings.items():
+        if name == 'timestamp':
```

Comment:
You should pop() above instead IMHO
e.g. not `timestamp = codebase.timings.get('timestamp',None)` but instead `timestamp = codebase.timings.pop('timestamp',None)`

Reply:
But this should not be in timings... but a proper attribute, so this is moot.